Assets/Scripts/Framework/Gameplay/LaneSystem/Note.cs

```diff
@@ -60,10 +82,27 @@ public void SetNoteValues(Lane lane,Vector2 initialPosition, Vector2 targetPosit
             transform.position = _initialPosition;
         }
 
+        /// <summary>
+        /// Instantiates a DeadNote and destroys this Note.
+        /// </summary>
+        public void HitNote()
+        {
+            GameObject deadNote = Instantiate(SessionValues.Instance.deadNote);
+
+            deadNote.GetComponent<DeadNote>().SetDeadNoteValues(_deathSprite, gameObject.transform);
+
+            Destroy(gameObject);
+        }
+
         private void Update()
         {
             if (SessionValues.Instance.paused || _laneID == -1)
+            {
+                _animator.speed = 0;
                 return;
+            }
+            else
+                _animator.speed = _animationSpeed / SessionValues.Instance.travelTime;
```

Comment:
Remove unnecessary else statement
```suggestion

            _animator.speed = _animationSpeed / SessionValues.Instance.travelTime;
```